poker/utils/action.py

```diff
@@ -0,0 +1,32 @@
+from poker.utils.chip import GameStack
+
+
+class Action:
+
+    CHECK = "check"
+    RAISE = "raise"
+    CALL = "call"
+    FOLD = "fold"
+
+    def __init__(self, game_stack: GameStack, player: any = None) -> None:
+        self.game_stack = game_stack
+        self.person = player
+
+    def check(self) -> None:
```

Comment:
Still to be implemented?

Reply:
I removed this method. It will be removed in a future PR.